README.md

```
@@ -1,10 +1,58 @@
+- It queries the database for all the traces that belong to this deployment.
+- It provides a user interface to select system level traces.
+- It visualizes the system level trace in the canvas using React Flow.
+- It visualizes the input/output variables in the system level trace.
+- It visualizes the input/output variables for the selected node in the system level trace.
+- It provides a user interface to filter through the traces with key values extracted from the trace input. 
+
+# Providing feedback
+
+You can use GitHub issues to [report a bug][bug-report] or [request a feature][feature-req].
 
 [demo-url]: https://vishalpalaniappan.github.io/automated-system-viewer/
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "demo-url"

(MD053, link-image-reference-definitions)